packages/common/src/index.ts

```diff
@@ -1225,27 +1225,41 @@ export function formatValue<T>(
     }
 }
 
-export function getFieldMap(explore: Explore): Record<string, CompiledField> {
+export function getFieldMap(
+    explore: Explore,
+    additionalMetrics: AdditionalMetric[] = [],
+): Record<string, Pick<CompiledField, 'format' | 'round'>> {
+    const additionalMetricFields = additionalMetrics.reduce(
+        (acc, additionalMetric) => ({
+            ...acc,
+            [fieldId(additionalMetric)]: {
```

Comment:
so you are applying the same format as the original field of the original metric, right ?

Reply:
No! The `additionalMetrics` have unique ids. It's just `{table}_{name}`